torch/nn/modules/_functions.py

```diff
@@ -196,3 +196,13 @@ def backward(ctx, grad_output):
                 accum_ratio.add_(paddded_ratio[c], alpha=-1)
 
         return grad_input, None, None, None, None
+
+class BackwardHookFunction(torch.autograd.Function):
+    @staticmethod
+    def forward(ctx, *args):
```

Comment:
Why not *args, **kwargs?

Reply:
kwargs are not allowed for custom Functions.